tests/llmcompressor/transformers/kv_cache/configs/kv_cache/phi3.yaml

```diff
@@ -0,0 +1,3 @@
+cadence: "commit"
+test_type: "regression"
+model: "microsoft/Phi-3-mini-4k-instruct"
```

Comment:
we only run e2e tests on a nightly cadence.
It should ideally be a larger model

Reply:
Nighly ok. 
Didnt we want a smaller model for deepseek?